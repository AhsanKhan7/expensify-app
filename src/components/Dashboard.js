import React from 'react'
import ExpenseList from './expensesList'
import ExpenseListFilters from './ExpenseListFilters'

const dashboardPage = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
)

export default dashboardPage