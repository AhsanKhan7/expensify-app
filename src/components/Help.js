import React from 'react'

const help = () => (
    <div>
        <p>this is help component</p>
    </div>
)

export default help