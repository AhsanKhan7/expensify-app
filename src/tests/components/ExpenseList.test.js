import React from 'react'
import { shallow } from 'enzyme'
import { ExpenseList } from '../../components/expensesList'
import expenses from '../fixtures/expense'

test('should expense list render', () => {
    const wrapper = shallow(<ExpenseList expenses={expenses}/>)
    expect(wrapper).toMatchSnapshot() 
})

test('should expense list render 2', () => {
    const wrapper = shallow(<ExpenseList expenses={[]}/>)
    expect(wrapper).toMatchSnapshot() 
})
