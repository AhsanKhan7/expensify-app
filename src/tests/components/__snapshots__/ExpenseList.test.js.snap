// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should expense list render 1`] = `
<div>
  <ExpensesListItem
    amount={195}
    createdAt={0}
    description="Gum"
    id="1"
    key="1"
    note=""
  />
  <ExpensesListItem
    amount={109500}
    createdAt={-345600000}
    description="Debit card"
    id="2"
    key="2"
    note=""
  />
  <ExpensesListItem
    amount={4500}
    createdAt={345600000}
    description="Rent"
    id="3"
    key="3"
    note=""
  />
</div>
`;

exports[`should expense list render 2 1`] = `
<div>
  <p>
    No expenses
  </p>
</div>
`;
